resources/assets/lib/app-deps.ts

```diff
@@ -38,3 +39,7 @@ window.LangMessages ??= {};
 window.Lang = new Lang({ messages: window.LangMessages });
 window.moment = moment;
 window.Turbolinks = Turbolinks;
+
+mobxConfigure({
```

Comment:
is this even the correct place to put this :thinking:

Reply:
as long as it runs before anything else mobx does 👀